tests/nvapi_d3d.cpp

```diff
@@ -56,3 +56,61 @@ TEST_CASE("D3D methods succeed", "[.d3d]") {
         REQUIRE(NvAPI_D3D1x_GetGraphicsCapabilities(&unknown, NV_D3D1x_GRAPHICS_CAPS_VER+1, reinterpret_cast<NV_D3D1x_GRAPHICS_CAPS*>(&caps)) == NVAPI_INCOMPATIBLE_STRUCT_VERSION);
     }
 }
+TEST_CASE("Reflex methods", "[.d3d]"){
+    UnknownMock unknown;
+    auto dxgiFactory = std::make_unique<DXGIFactory1Mock>();
+    auto vulkan = std::make_unique<VulkanMock>();
+    auto nvml = std::make_unique<NvmlMock>();
+    auto lfx = std::make_unique<LfxMock>();
+    DXGIDxvkAdapterMock adapter;
+    DXGIOutputMock output;
+
+    auto e = ConfigureDefaultTestEnvironment(*dxgiFactory, *vulkan, *nvml, *lfx, adapter, output);
+
+    SECTION("Enabling Reflex does not return OK when LFX is unavailable") {
+        SetupResourceFactory(std::move(dxgiFactory), std::move(vulkan), std::move(nvml), std::move(lfx));
+        REQUIRE(NvAPI_Initialize() == NVAPI_OK);
+
+        NV_SET_SLEEP_MODE_PARAMS params {};
+        params.version = NV_SET_SLEEP_MODE_PARAMS_VER;
+        params.bLowLatencyMode = true;
+        REQUIRE(NvAPI_D3D_SetSleepMode(&unknown, &params) != NVAPI_OK);
```

Comment:
Could you please do this for all three methods? The require can be changed to `== NVAPI_NO_IMPLEMENTATION`.